.github/workflows/perlcritic.yml

```diff
@@ -12,10 +12,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-
-    - name: "Run Perl::Critic" 
```

Comment:
There again could we go for a Make implementation using docker ? It will enable running the check locally to reproduce errors.

Reply:
Adding it to Make seems like a good idea! Especially since I can't seem to get GitHub action to work.